services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/nodeports.py

```diff
@@ -261,7 +281,7 @@ async def _get_data_from_port(
             if not downloaded_file or not downloaded_file.exists():
                 # the link may be empty
                 # remove files all files from disk when disconnecting port
-                logger.debug("removing contents of dir %s", final_path)
+                _logger.debug("removing contents of dir %s", final_path)
```

Comment:
this log should go in remove_directory as with log_context no?

Reply:
no that's a utility function. But I can wrap it in a log_context.